python/dgl/transform.py

```diff
@@ -43,8 +43,6 @@
     'to_block',
     'to_simple',
     'to_simple_graph',
-    'in_subgraph',
-    'out_subgraph',
     'as_immutable_graph',
     'as_heterograph']
```

Comment:
Can we add a TODO at the beginning: remove `copy_ndata` and `copy_edata` arguments?  Since copy-on-read is implemented I don't think they are necessary any more.

Reply:
I scanned the functions that have `copy_ndata` and `copy_edata`.
- `to_bidirected`
- `add_reverse_edges`
- `reverse`
- `to_block`
- `to_simple`

I feel many of their `copy_xdata` semantics are quite specific to the operator semantics. Maybe you can have a discussion with @classicsong and @mufeili on which arguments should be kept?

Reply:
I feel no function would require `copy_ndata` and `copy_edata` since they are all `index_copy`ing from the original tensor.